progress/bar.go

```diff
@@ -50,44 +57,49 @@ func (b *Bar) String() string {
 
 	var pre, mid, suf strings.Builder
 
-	if b.message != "" {
+	if len(b.message) > 0 {
 		message := strings.TrimSpace(b.message)
 		if b.messageWidth > 0 && len(message) > b.messageWidth {
 			message = message[:b.messageWidth]
 		}
 
 		fmt.Fprintf(&pre, "%s", message)
-		if b.messageWidth-pre.Len() >= 0 {
-			pre.WriteString(strings.Repeat(" ", b.messageWidth-pre.Len()))
+		if padding := b.messageWidth - pre.Len(); padding > 0 {
+			pre.WriteString(strings.Repeat(" ", padding))
 		}
 
 		pre.WriteString(" ")
 	}
 
 	fmt.Fprintf(&pre, "%3.0f%% ", math.Floor(b.percent()))
-	fmt.Fprintf(&suf, "(%s/%s", format.HumanBytes(b.currentValue), format.HumanBytes(b.maxValue))
 
-	stats := b.Stats()
-	rate := int64(stats.rate)
-	if rate > 0 {
-		fmt.Fprintf(&suf, ", %s/s", format.HumanBytes(rate))
-	}
+	suf.WriteRune('(')
 
-	fmt.Fprintf(&suf, ")")
+	rate := b.rate()
+	remaining := time.Duration(int64(float64(b.maxValue-b.currentValue)/rate)) * time.Second
 
-	elapsed := time.Since(b.started)
-	if b.percent() < 100 && rate > 0 {
-		fmt.Fprintf(&suf, " [%s:%s]", elapsed.Round(time.Second), stats.remaining)
-	} else {
-		fmt.Fprintf(&suf, "        ")
+	progress := format.HumanBytes(b.maxValue)
```

Comment:
This seems to show the following on initial render:
<img width="1185" alt="Screenshot 2023-11-20 at 5 08 49 PM" src="https://github.com/jmorganca/ollama/assets/251292/c0271919-f49a-45d6-b218-68300d0c6649">

I think we'd want to show the progress here (x/y)